spec.emu

```diff
@@ -281,7 +283,18 @@ contributors: Ujjwal Sharma, Younies Mahmoud
                 1. Perform ! CreateDataPropertyOrThrow(_nfOpts_, *"minimumFractionDigits"*, 𝔽(_durationFormat_.[[FractionalDigits]])).
               1. Perform ! CreateDataPropertyOrThrow(_nfOpts_, *"roundingMode"*, *"trunc"*).
               1. Set _done_ to *true*.
-          1. If _value_ is not 0 or _display_ is not *"auto"*, then
+          1. Let _displayRequired_ be *"false"*.
+          1. Let _hoursStyle_ be _durationFormat_.[[HoursStyle]].
+          1. If _unit_ is *"minutes"*, then
+              1. If _hoursStyle_ is *"numeric"* or *"2-digit"*, then
+                1. Let _hoursDisplay_ be _durationFormat_.[[HoursDisplay]].
+                1. Let _hoursValue_ be _durationFormat_.[[HoursValue]].
```

Comment:
There is no such thing call  _durationFormat_.[[HoursValue]] in this spec.  
In https://tc39.es/proposal-intl-duration-format/#sec-properties-of-intl-durationformat-instances Intl.DurationFormat instances does not have   internal slots  named [[HoursValue]] nor [[SecondsValue]]